src/gui/properties/pieceavailabilitybar.cpp

```diff
@@ -158,6 +158,10 @@ void PieceAvailabilityBar::clear()
 
 QString PieceAvailabilityBar::simpleToolTipText() const
 {
-    return tr("White: Unavailable pieces") + '\n'
-           + tr("Blue: Available pieces") + '\n';
+    const QString colorBoxBorderColor = palette().color(QPalette::ToolTipText).name();
```

Comment:
This color is common to all `PiecesBar` subclasses, isn't it? Then I would make it accessible via its interface like other such colors (e.g. `backgroundColor()`).

Reply:
I think this is not a property of Pieces Bar so I would like to leave it as it is.

Reply:
What is the difference with the other colors that are set there?